syntax = "proto2";

package com.scalableminds.webknossos.datastore;

import "geometry.proto";

message EditableMappingProto {
  required string baseMappingName = 1;
  required int64 createdTimestamp = 2;
  required int64 largestAgglomerateId = 3;
}
